apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1851,6 +1851,10 @@ curr_height() ->
     {_, Height} = curr_hash_and_height(),
     Height.
 
+curr_protocol() ->
```

Comment:
you saved 3 letters ;)

Reply:
there is also `curr_height` so didn't want to break function naming consensus :)